### PR TITLE
php-cs-fixer 2.11.1 (new formula)

### PR DESCRIPTION
**Pull request by javian**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
The version line needed adding since otherwise audit said `* Stable: version (cs) is set to a string without a digit` - should this be addressed in the version detection code perphaps ?

### CODE REVIEW

**Comment by javian** [comment]
Supplementary PR in homebrew/php for merging https://github.com/Homebrew/homebrew-php/pull/4820

**Comment by fxcoudert** [comment]
On Sierra it fails with:

```
==> /usr/local/Cellar/php-cs-fixer/2.11.1/bin/php-cs-fixer fix test.php
Loaded config default.

Fixed all files in 0.085 seconds, 11.250 MB memory used

Files that were not fixed due to errors reported during linting before fixing:
   1) /private/tmp/php-cs-fixer-test-20180325-41168-1kx9m0v/test.php
Error: php-cs-fixer: failed
<false> is not true.
```

On El Capitan, it fails with:

```
==> /usr/local/Cellar/php-cs-fixer/2.11.1/bin/php-cs-fixer fix test.php
PHP needs to be a minimum version of PHP 5.6.0 and maximum version of PHP 7.2.*.
```

File: Formula/php-cs-fixer.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  depends_on "php" if MacOS.version <= :el_capitan
+
+  def install
+    bin.install "php-cs-fixer.phar" => "php-cs-fixer"
+  end
+
+  test do
+    (testpath/"test.php").write <<~EOS
+      <?php $a = new    class(2){};
```
Comment (kabel) [inline Formula/php-cs-fixer.rb]: Can't use anonymous classes with PHP 5.6, which is what Sierra is picking up by default.

Comment (javian) [reply Formula/php-cs-fixer.rb]: Thanks kabel, I used something "simpler" now.

**Comment by ilovezfs** [comment]
Thanks again @javian!

**Pull request merged**